app/api/client/create/route.ts

```diff
@@ -0,0 +1,78 @@
+import dbConnect from "@/app/lib/dbConnect";
+import mongoose from "mongoose";
+import Client from "@/app/models/Client";
+import { ObjectId } from "mongoose";
+import bcrypt from "bcrypt";
+
+interface ClientModelInterface {
+  _id: string;
+  image: string;
+  name: string;
+  email: string;
+  password: string;
+  phone: string;
+  address: object;
```

Comment:
Change `address: object;` → `address?: object;`

The reason for this is because `address` is not required in the Model's `Schema`.